Cargo.toml

```
@@ -38,7 +38,7 @@ slab = "0.4.2"
 socket2 = { version = "0.3.12", features = ["pair", "unix"] }
 
 [dependencies.tokio]
-version = "0.2.19"
+version = "0.2.13"
```

```suggestion
version = "0.2"
```

Why not go all the way and allow users *any* 0.2.x version?

---

Good one; updated!